src/lib/useKeyfileGenerator.js

```diff
@@ -32,9 +32,12 @@ export default function useKeyfileGenerator(manualNetworkSeed) {
   const [code, setCode] = useState(false);
 
   const _point = need.point(pointCursor);
-  const _details = need.details(getDetails(_point));
+  const _details = getDetails(_point);
 
-  const networkRevision = convertToInt(_details.keyRevisionNumber, 10);
+  const networkRevision = _details.matchWith({
+    Just: ({ value }) => convertToInt(value.keyRevisionNumber, 10),
+    Nothing: () => -1,
```

Comment:
The value `-1` never naturally pops up for these. To avoid stubbing our toes on edge cases, should we maybe use `0` here instead?